search/search-domain/src/main/java/io/camunda/search/filter/UserTaskFilter.java

```diff
@@ -8,23 +8,26 @@
 package io.camunda.search.filter;
 
 import static io.camunda.util.CollectionUtil.addValuesToList;
+import static io.camunda.util.CollectionUtil.collectValues;
 import static io.camunda.util.CollectionUtil.collectValuesAsList;
 
+import io.camunda.util.FilterUtil;
 import io.camunda.util.ObjectBuilder;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 
-public final record UserTaskFilter(
+public record UserTaskFilter(
```

Comment:
💭 Same as in https://github.com/camunda/camunda/pull/24172, we might as well add operation awareness to all fields (except `boolean`s) already.

Reply:
I'll add them in a follow-up ticket.